css/properties/hyphens.json

```diff
@@ -16,7 +16,8 @@
             ],
             "chrome": [
               {
-                "version_added": "55"
+                "version_added": "55",
+                "notes": "<code>auto</code> value is only supported on macOS."
```

Comment:
`auto` is also supported on Android, which would also apply to `webview_android`. https://www.chromestatus.com/features/5642121184804864

Reply:
I wasn't sure on how to properly represent that, since `chrome` refers to the desktop editions (Windows, macOS, Linux), and `chrome_android is specific to the Android version.  I'm also not certain if Firefox, IE/Edge, and Safari support the `auto` value, and if so, which platforms.

Probably better to form some additional research on the matter before merging, haha.  :stuck_out_tongue:  I'll look into it a little more and push a commit when I have more info!

Reply:
This is too unusual of a condition to ask for affordance in the schema. A note is appropriate.

Reply:
It looks like the `auto` value is indeed supported on other browsers (tested on Firefox and Edge on macOS and Windows).  Thus, I believe this classifies as partial implementation for Chrome, especially since the `auto` value is [a part of the W3C specification](https://www.w3.org/TR/css-text-3/#hyphenation).  Updated commit coming up to reflect this information!